File: behavioral/python/mediator/Tablet.py
from Device   import Device

class Tablet(Device):

    def __init__(self, mediator):
        Device.__init__(self, mediator)
        self._mediator.setTablet(self)

    def runTest(self) ->None:
        print("Tablet test started.")
        self._mediator.runTablet()

    def endTest(self) ->None:
        print("Tablet test over.")

File: behavioral/Java/mediator/src/Tablet.java
public class Tablet extends Device{
    public Tablet(Mediator mediator) {
        super(mediator);
        this.mediator.setTablet(this);
    }

    @Override
    public void runTest() {
        System.out.println("Tablet test started.");
        this.mediator.runTablet();
    }

    @Override
    public void endTest() {
        System.out.println("Tablet test over.");
    }
}
